index.js

```diff
@@ -104,6 +104,12 @@
       swaggerObject = createSwaggerObject(loopbackApplication, opts);
     });
 
+  // listening to remoteMethodDisabled event for updating the swaggerObject
+  // when a remote method is disabled and remote it from the Swagger UI.
+    loopbackApplication.on('remoteMethodDisabled', function() {
```

Comment:
@superkhau  It seems like this whole file uses this style of indentation. I'm not sure it's the convention or not, so, I just followed. Should I change the indentation of the whole file?